CoordgenFragmentBuilder.cpp

```diff
@@ -1035,6 +1035,12 @@ void CoordgenFragmentBuilder::simplifyRingSystem(
                         /* disable rings that share too many atoms */
                         n++;
                     }
+                    if (r->fusionAtoms.at(ringCounter).size() == 3 &&
```

Comment:
so...... We spit stuff apart and _then_ figure out if it matches any of the templates? Could we (in the future) match on the templates first, and then decide what to split on so that we don't need special logic for each new template?

Reply:
well, before matching templates we only remove rings that are easy to put back (originally only rings that share 2 atoms, now we do also 3 matching atoms, which was a problem for this template but is usually safe). If we matched templates before we'd need an awful lot of extra templates. We might consider matching templates before and after removing rings, but I really don't anticipate a lot of extra special-casing like this one